packages/cf-style-container/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "dependencies": {
     "css-color-function": "^1.3.0",
+    "fela": "^4.3.2",
```

Comment:
`cf-style-container` is a set of helpers that are used **by Fela components**. `cf-style-provider` is used **by consumers of cf-ui**. We definitely don't want to merge them because `cf-style-provider` has a lot of Fela dependencies (plugins, enhancers etc) that have no use for components. They don't overlap at all.